File: aa/PRAC/PERL/switch.pl
#!/usr/bin/perl
use strict;
use warnings;
use Switch;

my @arr = (10,20,30,40);
my $aa = 10;
my %hh = ('a',10,'b',20,'c',30);

switch($aa) {
	case  "a" { print "string a\n"; }

	case 10 { print "the value is 10\n"; }

	else   { print "case is not true\n"; }
}

File: aa/PRAC/PERL/switch.py
#!/usr/bin/python3


def SwitchExample(argument):
    switcher = {
        0: " This is Case Zero ",
        1: " This is Case One ",
        2: " This is Case Two ",
    }
    return switcher.get(argument, "nothing")


if __name__ == "__main__":
    #argument = 1
    argument = int(input())
    print (SwitchExample(argument))
